common/libs/nodes/infura/client.ts

```diff
@@ -2,7 +2,7 @@ import { randomBytes } from 'crypto';
 import RPCClient from '../rpc/client';
 
 export default class InfuraClient extends RPCClient {
-  public id(): string {
-    return `0x${randomBytes(5).toString('hex')}`;
+  public id(): number {
+    return parseInt(randomBytes(5).toString('hex'), 16);
```

Comment:
👍

Reply:
So infura takes a number for ID but RPC node takes a string?

Reply:
Yes, oddly

Reply:
Yeah, RpcNodes for our apis take a hex string, but the Infura nodes need a integer or it replies with a Golang error about parsing hex.